src/panels/lovelace/editor/process-view-visibility.ts

```diff
@@ -0,0 +1,24 @@
+import { ShowViewConfig } from "../../../data/lovelace";
+import { User } from "../../../data/user";
+
+export function processViewVisibility(
+  visible: boolean | ShowViewConfig[],
+  users?: User[]
+): ShowViewConfig[] {
+  if (typeof visible === "boolean") {
+    const allVisible = visible as boolean;
```

Comment:
Don't do this, what if I want the current three users to have access, but later add a fourth that I don't automatically want to give access?

Instead, add an extra global toggle that will hide the user selection.

Reply:
This will be done only once. Visible can be a boolean value, but when saving I'm always storing a list of users ids. This way, when you add a new user he won't see the view (because his id won't be on the list)